README.md

```diff
@@ -1,9 +1,15 @@
-Follow steps below to use:
 
-Edit any details to customise the environment as you require in ansible-hosts-multimaster.
-
-Run:
+To execute the playbooks and set up your Openshift environment, run the following:
 
 ```
 ./deploy-openshift.sh
 ``` 
+
+The environment definition is in the file "/etc/ansible/group_vars/all.yml". This file is written by the ansible-heat code.
```

Comment:
Should this be openshift-heat code instead of ansible-heat?

Reply:
fixed